salvo/src/lib/test_source_manager.py

```diff
@@ -291,6 +291,66 @@ def test_find_all_images_from_specified_tags_fail(mock_source_tree):
   assert str(source_error.value) == \
     "No images are specified or able to be built from the control document"
 
+def test_find_image_single():
+  """Verify that we can just find specific image with test_single_image enabled"""
+  job_control = proto_control.JobControl(
+      remote=False,
+      scavenging_benchmark=True,
+  )
+
+  _generate_default_benchmark_images(job_control)
+
+  # Add an envoy image and specify additional versions to test
+  job_control.images.envoy_image="envoyproxy/envoy:v1.16.0"
+
+  # Specify test_single_image as true
+  job_control.images.test_single_image=True
```

Comment:
This looks mis-formatted, should we have spaces around the equal sign? This applies to multiple locations.

I know we had some auto formatting tools in this repository, could you try to find and execute them before we do the next review round?

Reply:
@mum4k I create another PR #134  to fix python files format, because it's large change which may confuse your review of this PR, can you merge it first? Then I will rebase this PR on that. Thanks!

Reply:
Thank you for making the review easier. Reviewed and merged. Please merge here and rerun the formatting tools on files modified in this PR.

Reply:
Done, please review again thanks!